peer-monitoring-service/client/src/tests/single_peer.rs

```diff
@@ -288,7 +293,21 @@ async fn test_latency_ping_failures() {
         MockMonitoringServer::new(vec![network_id]);
 
     // Create a node config where only latency pings refresh
-    let node_config = config_with_latency_ping_requests();
+    let node_config = NodeConfig {
```

Comment:
You could probably make this a little cleaner but just modifying the config directly? e.g.,
```
// Create a node config where only latency pings refresh
let mut node_config = config_with_latency_ping_requests();

// Ensure we tolerate enough ping failures to pass the test
node_config.peer_monitoring_service.latency_monitoring.max_latency_ping_failures = 20;
```